### ISSUE TITLE
Zero-shot grounding

### ISSUE THREAD

**Issue by HieuPhan33** [laihaoran/CARZero#3]
Hi,

Thanks for sharing your amazing work.
Could I ask if you can share the code to evaluate zero-shot grounding?

**Comment by laihaoran** [comment]
Yes, I will share the code to evaluate zero-shot grounding soon.

**Comment by bmyoon1234** [comment]
Not shared yet? @laihaoran
